src/Microsoft.Azure.SignalR.AspNet/Middleware/NegotiateMiddleware.cs

```diff
@@ -78,8 +78,18 @@ public override Task Invoke(IOwinContext owinContext)
             {
                 return ProcessNegotiationRequest(owinContext, context);
```

Comment:
Could you also create an issue describing the problem?

Reply:
Add an issue:
https://github.com/Azure/azure-signalr/issues/1236